### PR TITLE
Add steps to cherry-pick a commit

### PR DESCRIPTION
**Pull request by daxmc99**

### CODE REVIEW

**Comment by daxmc99** [comment]
@rata I will update this to specify that the cherry-pick should happen right after the merge (I performed the cherry-pick right after the merge btw). 

I will wait to merge until we get @russellb's maintainer's PR in.

File: website/content/community/release-process.md [hunk 1]
```diff
@@ -7,12 +7,25 @@ This is the process to follow to make a new release. This should
...
+
+
 ### Check the issue tracker milestone
 
 Are you actually ready to release? Check the milestone on github and
 verify that all its issues are closed. If there are open issues,
 you'll have to either resolve them, or bump to the next version.
 
+
+### ### Cherry-pick relevent commits
```
Comment (johananl) [inline website/content/community/release-process.md:19]:
```suggestion
### Cherry-pick relevent commits
```

File: website/content/community/release-process.md [hunk 2]
```diff
@@ -7,12 +7,25 @@ This is the process to follow to make a new release. This should
...
+
+MetalLB uses release branches to track releases. Relevant commits should be cherry-picked onto the release branch.
+For example,
```
Comment (johananl) [inline website/content/community/release-process.md:22]:
```suggestion
For example:
```

File: website/content/community/release-process.md [hunk 3]
```diff
@@ -7,12 +7,25 @@ This is the process to follow to make a new release. This should
...
+```
```
Comment (johananl) [inline website/content/community/release-process.md:23]: I'd add an empty line before the snippet.

Comment (rata) [reply website/content/community/release-process.md:24]: A new line has been added, marking as solved

**Comment by rata** [comment]
> I will wait to merge until we get @russellb's maintainer's PR in.

@daxmc99 Not sure I follow, this isn't based on that, right? Not sure I follow why to wait to the other PR to get in if this is not based on that. It fits better here?

Anyways, wherever this ends up being, can you just amend all the commits and remove the merge commit? Just one commit for this simpler change seems enough :)

**Comment by russellb** [comment]
@daxmc99 just a reminder that this one is waiting on a rebase

**Comment by russellb** [comment]
I rebased and squashed this so I could merge it.

**Pull request merged**